pwnlib/tubes/ssh.py

```diff
@@ -554,6 +554,7 @@ def __init__(self, user=None, host=None, port=22, password=None, key=None,
             level: Log level
             cache: Cache downloaded files (by hash/size/timestamp)
             ssh_agent: If :const:`True`, enable usage of keys via ssh-agent
+            use_environment: If :const:`False`, disable usage of ~/.ssh/config and ~/.ssh/authorized_keys
```

Comment:
Can we change this to `isolated` or `ignore_config` or `ignore_dotfiles` per the main discussion?